Lib9c.Policy/Policy/MaxTransactionsPerSignerPerBlockPolicy.cs

```diff
@@ -17,7 +17,7 @@ private MaxTransactionsPerSignerPerBlockPolicy(int defaultValue)
         public static IVariableSubPolicy<int> Default =>
             new MaxTransactionsPerSignerPerBlockPolicy(int.MaxValue);
 
-        public static IVariableSubPolicy<int> Mainnet =>
+        public static IVariableSubPolicy<int> Odin =>
```

Comment:
Can we consider either raising or eliminating the SpannedSubPolicy value across all networks? The current maximum value of 4 is restricting the swift processing of our internal service's transactions.

Reply:
I think so. (I'm not a fan of discriminating internal v. public services, but it's why that we need to replace this policy to MEAD based tx control. 😅 )

But even if so, it might be large work for this PR.